Jenkinsfile

```diff
@@ -7,6 +7,7 @@ node {
    sh "docker-compose run web buildout"
 
    stage "Test"
+   sh "docker-compose run --rm web bin/flake8"
```

Comment:
```suggestion
   sh "docker-compose run --rm web bin/flake8 lizard_auth_client"
```

By specifying the subdirectory, you don't need to have a long list of excludes in the `setup.cfg` for flake8.

Reply:
Done!